ko/news/_posts/2024-04-23-ruby-3-0-7-released.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: news_post
+title: "Ruby 3.0.7 릴리스"
+author: "hsbt"
+translator: "shia"
+date: 2024-04-23 10:00:00 +0000
+lang: ko
+---
+
+Ruby 3.0.7이 릴리스되었습니다.
+
+이번 릴리스에는 보안 수정 사항이 포함되어 있습니다.
+자세한 내용은 아래 항목을 참조하세요.
+
+* [CVE-2024-27282: 정규표현식 검색의 임의의 메모리 주소 읽기 취약점]({%link ko/news/_posts/2024-04-23-arbitrary-memory-address-read-regexp-cve-2024-27282.md %})
+* [CVE-2024-27281: RDoc에서 .rdoc_options 사용 시의 RCE 취약점](https://www.ruby-lang.org/ko/news/2024/03/21/rce-rdoc-cve-2024-27281/)
+* [CVE-2024-27280: StringIO에서 버퍼 초과 읽기 취약점](https://www.ruby-lang.org/ko/news/2024/03/21/buffer-overread-cve-2024-27280/)
```

Comment:
These links can be `{%link ... %}` too, so `en` version also can get fixed. It's same with 3.1.5, 3.2.4, and 3.3.1 posts too.

Reply:
fixed 1e9d46c7044405b5e79d1475a0ea6b15189d9ec6

Reply:
and also en \w https://github.com/ruby/www.ruby-lang.org/pull/3231